.trivyignore

```diff
@@ -1,13 +1,5 @@
-# List any vulnerability that are to be accepted
-
-# See https://aquasecurity.github.io/trivy/v0.35/docs/vulnerability/examples/filter/
-
+# See https://aquasecurity.github.io/trivy/v0.35/docs/vulnerability/examples/filter/ 
 # for more details
-
-# e.g.
-
+# e.g. 
 # CVE-2022-3996
-
-# https://atlassian.thetradedesk.com/jira/browse/UID2-3300
-
 CVE-2022-29622
```

Comment:
Should we leave a comment for why we are ignoring or linking to a Jira ticket to address the vulnerability?

Reply:
@ssundahlTTD do you have the ticket number for this?

Reply:
https://atlassian.thetradedesk.com/jira/browse/UID2-2633 is what I'm working on for the other vulnerbailities